docs/releases/hotfix-release.md

```diff
@@ -101,7 +101,7 @@ As noted above, the challenges in creating a hotfix are unique to each attempt.
       If you are hotfixing a beta release, run the following command:
 
       ```bash
-      node ./common/scripts/force-build-flavor.mjs beta
+      node ./common/scripts/force-build-flavor.mjs beta-stable
```

Comment:
this should just be `beta` `beta-next` is for the UI lib version bumping which you are changing to `beta` above.

Reply:
This updates the matrix.json with `beta` instead of `beta-release` which seems incorrect for a hot fix beta branch?